modules/sde-submodules/batch/batch.md

```diff
@@ -5,17 +5,17 @@
 This module use for Batch submodel specification and descriptors. It's contain the codes related to Batch to validate, parse and transfer data for DigitalTwins and EDC to create aspect twins and data offer.
 
 ---
-#### Version: 1.0.0
-#### Batch Aspect Model URN: urn:bamm:com.catenax.batch:1.0.0#Batch
-#### Semantic Id: urn:bamm:com.catenax.batch:1.0.0
+#### Version: 1.0.2
+#### Batch Aspect Model URN: urn:bamm:io.catenax.batch:1.0.2#Batch
+#### Semantic Id: urn:bamm:io.catenax.batch:1.0.2
 ---
 
 ### Schema
 
 Please find below links for schema details:
 
-- https://portal.int.demo.catena-x.net/semantichub/urn%3Abamm%3Aio.catenax.batch%3A1.0.0%23Batch
-- blob:https://portal.int.demo.catena-x.net/10dadf03-8062-4369-87c8-6f8f570df2c4
+- https://portal.int.demo.catena-x.net/semantichub/urn%3Abamm%3Aio.catenax.batch%3A1.0.2%23Batch
+- blob:https://portal.int.demo.catena-x.net/07fc4b99-452f-4afd-bd41-fdf4c1d96318
 - https://github.com/catenax-ng/product-dft-backend/blob/modules-branch/modules/sde-submodules/batch/src/main/resources/batch.json
```

Comment:
could you modify the links to eclipse tractus-x ?

Reply:
sure,I will modify.